datenguide_python/query_builder.py

```diff
@@ -20,32 +25,61 @@ class Field(NamedTuple):
         If the filter is not set, then only the summed result is returned.)
     """
 
-    name: str
-    subfields: list
-    args: Optional[Dict[str, Union[str, List[str]]]] = None
+    def __init__(
+        self,
+        name: str,
+        fields: List[Union[str, object]] = [],
```

Comment:
Use `Union[str,"Field"]`. Note the quotes! This is the way to use classes as types within their own bodies.

Reply:
ah okay, this answers my question from above i guess ;)

Reply:
Ok I also answered it again above before I arrived at this reply.